.gitignore

```diff
@@ -1,4 +0,0 @@
-_site/
-.sass-cache/
-.jekyll-metadata
-*~
```

Comment:
I didn't think to only create the branch from the master branch. I'll put that in my git help notes. 

I think my master branch is ahead was because of screwing up before. I initially created the radioactive-decay content in the master branch, so then I deleted all that and made the actual radioactive-decay branch. 

I probably would need help fixing everything because on my own, I would just end up deleting everything and forking necx-org again and redoing everything from scratch.